pycfmodel/utils.py

```diff
@@ -12,3 +12,19 @@
 CONDITIONS OF ANY KIND, either express or implied. See the License for the
 specific language governing permissions and limitations under the License.
 """
+from typing import Any
+
+from .constants import IMPLEMENTED_FUNCTIONS, CONDITION_FUNCTIONS
+
+
+def is_resolvable_dict(value: Any) -> bool:
+    return isinstance(value, dict) and len(value) == 1 and next(iter(value)) in IMPLEMENTED_FUNCTIONS
+
+
+def is_conditional_dict(value: Any) -> bool:
+    if isinstance(value, dict):
+        for func in value.keys():
+            if not any(f in func for f in CONDITION_FUNCTIONS):
+                return False
+        return True
+    return False
```

Comment:
Are there tests for this function? I'm not 100% sure it does what's expected.

Reply:
I will add tests